aten/src/ATen/core/IList.h

```diff
@@ -0,0 +1,512 @@
+#pragma once
```

Comment:
No changes needed here, but this is the sort of thing where I don't think stacking the PRs helps much, it would be better to merge the PRs into one mega PR so I can see where everything is going to be finally.

Reply:
Hmmm. Is it because they are incremental PRs that overlap (on the code they are changing)? I thought this was exactly the type of changes where it made sense using a stack of PRs, since:
- There are less code to be reviewed at a time
- Makes it easier to merge and revert

On the other hand, it does feel like the code is always changing, though. But I thought the benefits outweighed the drawbacks. Anyway, I will try to think a bit more next time.

Reply:
It's definitely a delicate balance. On the one hand reducing the amount of code to review makes review easier. But if you ask me if I'd prefer one PR (200LOC), or two PRs (150LOC each) where the first PR takes a detour before getting to the final destination in the second PR, I'd prefer one PR, it's less work net for me and I don't have to spend time understanding an intermediate state that will immediately get moved away from.

That's not to say that intermediate step PRs don't have their place; for example when a change is too complex/risky to do in one go, taking detours is a good way to derisk. But if you introduce something new and then immediately change it next, well, that just suggests the diff stack is just in chronological order rather than ease for review. And sometimes it is possible to split up the diff for review, but it's not worth it (I'm allowed to say this, cuz I'm the reviewer :P ).

Reply:
That does make sense. I will make sure to take that into consideration next time.
Anyway, do you think I should I merge #67964 and #69606?

Reply:
It's up to you, we've reviewed these PRs now so it is mostly a matter of how to get these over the finish line one way or another.